.gitignore

```diff
@@ -10,6 +10,7 @@ rest-client.private.env.json
 # Default ignored files
 .idea/workspace.xml
 .idea/shelf
+.idea/deployment.xml
```

Comment:
Please, get rid of this change

Reply:
Why do you commit the .idea directory?

Reply:
it stores the project settings - like exclusion of vendor folders for example. We are open to any pull requests to it but it's better to do it in a separate PR